src/platform/api/api.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import { getEnv } from "../../env";
+import network from "../../network";
+import type { AppManifest, PlatformApi } from "../types";
+
+// expose a function to fetch data from the cdn (data from ledger-live-assets)
+// https://cdn.live.ledger.com/
+
+const basePlatformUrl = () => getEnv("PLATFORM_API_URL");
+const platformVersion = () => getEnv("PLATFORM_API_VERSION");
+
+async function fetchManifest(): Promise<AppManifest[]> {
+  const url = `${basePlatformUrl()}/v${platformVersion()}/data.json?t=${Date.now()}`;
```

Comment:
is there any reason we pass the time here? It doesn't make a lot of sense to me and it bypass CDN capability to cache which may not be a good idea.

Reply:
This was inspired by the Announcement api, so i thought there were reasons to do so.
I agree that it may bypass CDN's cache, i think it's intended to only bypass a local xhr cache.